vnext/Desktop.ABITests/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.1" targetFramework="native" />
-	<package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
+  <package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.1" targetFramework="native" />
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
+  <package id="XamlStraddle" version="0.0.3" targetFramework="native" />
```

Comment:
>XamlStraddle [](start = 15, length = 12)

The ABITests must not depend on XAML.

Reply:
Agree but it does today. Feel free to file/fix though :-)